src/host/libpixyusb_swig/get_frame.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
#!/usr/bin/python3

????

Reply:
I tried wrapping with python3 but get a segmentation fault when sending commands. I don't think it's worth the effort especially when we need a better camera platform anyways. The more I work with pixy the more I'm concerned about its limitations.

Reply:
ok